tests/test_check_runner.py

```diff
@@ -0,0 +1,145 @@
+"""Test suite for check_runner"""
+
+# pylint: disable=E1101, W0621, R0903
+
+import time
+from unittest.mock import call
+
+import pytest
+import cachetclient.cachet
+
+import check_runner as unit
+from monitoring_scripts.nagios_common import Codes
+from . import common
+
+
+class CompletedProcessMock():
+    """Mocks subprocess.CompletedProcess to test returncode"""
+
+    def __init__(self, returncode):
+        self.returncode = returncode
+
+
+@pytest.fixture(scope='module')
+def config(tmpdir_factory, base_url='http://some.url', api_key='namaste'):
+    """Fixture that generates a customisable config file"""
+
+    if base_url:
+        base_url_line = 'base-url: ' + base_url + '\n'
+    else:
+        base_url_line = ''
+    if api_key:
+        api_key_line = 'api-key: ' + api_key + '\n'
+    else:
+        api_key_line = ''
+
+    config_file = tmpdir_factory.mktemp('data').join('config.ini')
+    config_file.write('[Cachet]\n' + base_url_line + api_key_line)
```

Comment:
Was there an issue making `ConfigParser` work ?

Reply:
Since no heavy task was expected I decided to go with a straightforward `write`.

Reply:
Alright, I'd recommend to go straight to the `ConfigParser` path next time, if only to be consistent in the APIs we interact with. It's fine for now though.